lvsfunc/util.py

```diff
@@ -102,6 +103,45 @@ def get_prop(frame: vs.VideoFrame, key: str, t: Type[T]) -> T:
     return prop
 
 
+def select_frames(clips: Union[vs.VideoNode, List[vs.VideoNode]],
+                  indicies: Union[List[int], List[Tuple[int, int]]]) -> vs.VideoNode:
+    """
+    Select frames from one or more clips at specified indices.
```

Comment:
Might be more understandable to say something like `Builds a clip from a list of clips and frame numbers`

Reply:
Yeah I intended for this to be rewritten before merging. I still don't really like the description here, but I'm unsure how to nicely describe what this does in a short sentence.